src/connection/captcha.js

```diff
@@ -16,7 +16,9 @@ export function showMissingCaptcha(m, id, isPasswordless = false) {
 
   const captchaError = (
     captchaConfig.get('provider') === 'recaptcha_v2' ||
-    captchaConfig.get('provider') === 'recaptcha_enterprise'
+    captchaConfig.get('provider') === 'recaptcha_enterprise' ||
+    captchaConfig.get('provider') === 'hcaptcha' ||
+    captchaConfig.get('provider') === 'friendly_captcha' 
```

Comment:
Is it expected to result in `invalid_recaptcha` when using `hcaptcha` or `friendly_captcha` ?

Reply:
Yes, this is used to display the appropriate localized error message which is consistent for all intents and purposes. Although I don't love the naming, this would require changes to all localization files in i18n which is a rather extensive change.